prow/plugins/golint/golint.go

```diff
@@ -133,30 +161,62 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, ic github.Issue
 	if err != nil {
 		return err
 	}
+	oldComments, err := ghc.ListPullRequestComments(org, repo, ic.Issue.Number)
+	if err != nil {
+		return err
+	}
+	nps := newProblems(oldComments, problems)
 
 	// Respond.
 	var response string
 	var comments []github.DraftReviewComment
-	if len(problems) == 0 {
-		response = "no lint warnings"
-	} else {
-		response = fmt.Sprintf("%d warning(s)", len(problems))
-		for patchPath, problem := range problems {
+	for f, ls := range nps {
+		for l, p := range ls {
+			var body string
+			if p.Link == "" {
+				body = fmt.Sprintf("Golint %s: %s. %s", p.Category, p.Text, commentTag)
+			} else {
+				body = fmt.Sprintf("Golint %s: %s. [More info](%s). %s", p.Category, p.Text, p.Link, commentTag)
+			}
 			comments = append(comments, github.DraftReviewComment{
-				Path:     problem.Position.Filename,
-				Position: patchPath,
-				Body:     problem.Text,
+				Path:     f,
+				Position: l,
+				Body:     body,
 			})
 		}
 	}
 
+	numOldProblems := numProblems(problems)
```

Comment:
Nit: This is really the total number of problems, not the number of old problems.